pfcpiface/bess.go

```diff
@@ -1146,6 +1150,12 @@ func (b *bess) addSliceMeter(ctx context.Context, done chan<- bool, meterConfig
 			ebs = 0 // Unused
 		}
 
+		log.Println("uplink slice :")
+		log.Println("cir : ", cir)
+		log.Println("pir : ", pir)
+		log.Println("cbs : ", cbs)
+		log.Println("pbs : ", pbs)
+		log.Println("ebs : ", ebs)
```

Comment:
Single line log.Traceln(). Same for below.

Reply:
done